src/main/kotlin/zhi/yest/vk/friendfinder/vk/GroupsService.kt

```diff
@@ -0,0 +1,8 @@
+package zhi.yest.vk.friendfinder.vk
+
+import org.springframework.security.oauth2.core.user.OAuth2User
+import zhi.yest.vk.friendfinder.domain.Group
+
+interface GroupsService {
+    suspend fun findById(groupId: String, oAuth2User: OAuth2User): Group
```

Comment:
can we make oAuth2User: OAuth2User a request scoped/session scoped bean and inject it?